packages/embark/src/lib/modules/blockchain_listener/index.js

```diff
@@ -115,6 +148,19 @@ class BlockchainListener {
     }
     this.events.once('blockchain:devtxs:ready', () => { this.devTxs.sendTx(cb); });
   }
+
+  _saveProxyLog(log) {
+    this.writeProxyLogFile.push(log);
+  }
+
+  _readProxyLogs() {
+    this.fs.ensureFileSync(this.proxyLogFile);
+    try {
+      return JSON.parse(this.fs.readFileSync(this.proxyLogFile));
+    } catch (_error) {
+      return {};
```

Comment:
Do we want to print the error?
Also, returning `{}` will cause the next write to override the log file. It can be good in the case of a "corrupted" file, but could be not nice.

Reply:
Hmm, I actually took this from the logic in `console_listener` and didn't look in to that much. But i believe that it's not a massive deal as it would only happen in the case of a corrupted file as you mentioned, and in that case would correct itself.